test/unit/media_timeline_utils.spec.js

```diff
@@ -0,0 +1,1226 @@
+/* eslint-disable */
```

Comment:
Sorry -- there's some cruft in `test/unit/`, but really nothing belongs there.

Please move this file so it becomes `src/app/components/media/MediaTimeline.test.js`.

Also -- as always, please don't disable eslint. Change this line to `/* globals describe, expect, it */`

Reply:
fixed, `./node_modules/.bin/jest src/app/components/media/MediaTimeline`